impeller/entity/entity_pass.cc

```diff
@@ -129,11 +129,13 @@ std::optional<Rect> EntityPass::GetElementsCoverage(
                                           Entity::RenderingMode::kSubpass);
         if (backdrop_filter) {
           auto backdrop_coverage = backdrop_filter->GetCoverage({});
-          backdrop_coverage->origin += result->origin;
           if (backdrop_coverage.has_value()) {
             if (unfiltered_coverage.has_value()) {
-              unfiltered_coverage = coverage->Union(*backdrop_coverage);
+              unfiltered_coverage = coverage.has_value()
```

Comment:
I might be missing something, but I don't see how `coverage.has_value()` could ever be true here.

Reply:
Also, this tests `backdrop_coverage.has_value()` after it already used the `->` operator on it...

Reply:
Is the bug here that the calculation should have been `unfiltered_coverage->Union(*backdrop_coverage)` all along?

Reply:
Fixed, yeah this was supposed to union with unfiltered_coverage.